lib/gap_intelligence/models/merchant_pricing_trend.rb

```diff
@@ -0,0 +1,9 @@
+module GapIntelligence
+  class MerchantPricingTrend < Record
+    attributes :values, :headers
+
+    def initialize(attrs = {})
+      @attributes = attrs
+    end
```

Comment:
here is doc for fetch method http://ruby-doc.org/core-2.2.0/Hash.html#method-i-fetch